libs/components/layout/src/lib/modules/fluid-grid/column.component.scss

```diff
@@ -10,41 +19,64 @@
   }
 }
 
-@mixin create_column_gutters($gutter-size) {
```

Comment:
These mixins got a little clunky in the token world because of the tokens using `s`, `m`, and `l` while the classes use the written out versions. I just decided to axe the mixins and just write out the styles as it is a little clearer what the end result is that way anyway. I can reconsider if you feel strongly about it.